ClosedXML_Sandbox/Program.cs

```diff
@@ -18,6 +18,10 @@ private static void Main(string[] args)
             Console.WriteLine();
 #endif
 
+            Console.WriteLine("Running {0}", nameof(PerformanceRunner.RunInsertTableWithStyles));
```

Comment:
No problem. It may be excluded at all - I used this benchmark to make sure the performance didn't get too worse. There is no need to preserve this test permanently.